### PR TITLE
Add circle using turtle

### LINKED ISSUE #2298: Want to add an animation of circle using turtle
**Describe the project you want to add with tech stack**

An animation of circle using Turtle library in Python

**Expected behavior**

Animates circle with an interesting texture.

**Screenshots (optional)**

https://github.com/pranjay-poddar/Dev-Geeks/assets/95526205/753320de-7700-4dd8-8ebe-e6f12639edc7

### PR DESCRIPTION
**Pull request by ahana02**
# Want to add an animation of circle using turtle #2298 

Close #2298 

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Comment by pranjay-poddar** [comment]
Resolve conflicts

**Comment by ahana02** [comment]
@pranjay-poddar I have resolved the conflict. Please add the `gssoc23` labels and merge.

**Comment by ahana02** [comment]
@pranjay-poddar resolved the conflict again, please add labels and merge it.

**Comment by ahana02** [comment]
@MohitGupta121 can I please know the reason of reverting this PR?

File: Python Projects/Circle using Turtle/assets/circle.mp4
Comment (MohitGupta121) [inline Python Projects/Circle using Turtle/assets/circle.mp4:1]: @ahana02 if you want labels and not revert then please make new PR to remove this video or use this video through link upload somewhere else and use here with that link. Thanks please do that.

Comment (ahana02) [reply Python Projects/Circle using Turtle/assets/circle.mp4:1]: @MohitGupta121 I have done the changes you asked for and created a new PR #3511 . Add GSSoC labels and merge it.

**Pull request merged**